packages/cli/test/dev/utils.js

```diff
@@ -347,6 +347,9 @@ function testFixtureStdio(
               ? ['--scope', process.env.VERCEL_TEAM_ID]
               : []),
             'deploy',
+            ...(process.env.VERCEL_CLI_VERSION
+              ? ['--build-env', process.env.VERCEL_CLI_VERSION]
```

Comment:
```suggestion
              ? ['--build-env', `VERCEL_CLI_VERSION=${process.env.VERCEL_CLI_VERSION}`]
```

Should this be set like this?